newIDE/app/src/ResourcesList/ResourceUtils.js

```diff
@@ -0,0 +1,10 @@
+export const createOrUpdateResource = (project, gdResource, resourceName) => {
+    const resourcesManager = project.getResourcesManager();
+    if (resourcesManager.hasResource(resourceName)) {
+        resourcesManager.removeResource(resourceName)
+    };
+    gdResource.setFile(resourceName);
+    gdResource.setName(resourceName);
+    resourcesManager.addResource(gdResource);
+    gdResource.delete();
+};
```

Comment:
You can configure your editor to add an empty line at the end of files. The reason for this is quite funny: https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-file

Reply:
I didnt know that, thank you for sharing :)